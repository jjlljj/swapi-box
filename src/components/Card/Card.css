.card {
  background-color: #1A1A1A;
  box-sizing: border-box;
  color: #d2d2d2;
  max-height: 410px;
  min-height: 320px;
  margin: 20px;
  position: relative;
  text-align: center;
  width: 290px;
}

.card header {
  background-color: #1A529D;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 90px;
}

.card h3 {
  display: inline-block;
  font-size: 28px;
  letter-spacing: 4px;
  color: #feda4a;
  font-weight: 800;
  margin: 0 50px;
}

.card ul {
  margin: 30px 20px;
}

.card li {
  margin: 15px 20px;
  font-size: 18px;
}

.card button {
  background-color: transparent;
  border-radius: 50%;
  border: 1px solid #f2f2f2;
  color: #d2d2d2;
  font-size: 22px;
  font-weight: bold;
  height: 40px;
  padding: 0;
  position: absolute;
  right: 15px;
  top: 24px;
  width: 40px;
}

button:hover {
  cursor: pointer;
}

.card .favorite {
  color: #feda4a;
  border: 2px solid #feda4a;
}

