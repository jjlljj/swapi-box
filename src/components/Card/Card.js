import React from 'react';
import propTypes from 'prop-types';
import './Card.css';

const Card = () => {
  return (
    <div> i am card </div>
  );
};

export default Card;
