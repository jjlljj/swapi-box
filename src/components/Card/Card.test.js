/* eslint-disable */

import React from 'react';
import ReactDom from 'react-dom';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {

  it('should exist', () => {
  
  })

})
