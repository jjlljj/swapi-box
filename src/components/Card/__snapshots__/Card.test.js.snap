// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot when rendering a person card 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card
    addToFav={[Function]}
    card={
        Object {
            "Homeword": "Tatooine",
            "Population": "200000",
            "Species": "Human",
            "cardType": "people",
            "name": "Beru Whitesun lars",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h3>
                    Beru Whitesun lars
          </h3>
          <button
                    className=""
                    onClick={[Function]}
                    title="Add to Favorites"
          >
                    ★
          </button>
</header>,
        <ul>
          <li>
                    Homeword
                    : 
                    Tatooine
          </li>
          <li>
                    Population
                    : 
                    200000
          </li>
          <li>
                    Species
                    : 
                    Human
          </li>
</ul>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Beru Whitesun lars
</h3>,
            <button
              className=""
              onClick={[Function]}
              title="Add to Favorites"
>
              ★
</button>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Beru Whitesun lars",
            },
            "ref": null,
            "rendered": "Beru Whitesun lars",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "★",
              "className": "",
              "onClick": [Function],
              "title": "Add to Favorites",
            },
            "ref": null,
            "rendered": "★",
            "type": "button",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Homeword
              : 
              Tatooine
</li>,
            <li>
              Population
              : 
              200000
</li>,
            <li>
              Species
              : 
              Human
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Homeword",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Homeword",
              ": ",
              "Tatooine",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Population",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Population",
                ": ",
                "200000",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Population",
              ": ",
              "200000",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Species",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Species",
                ": ",
                "Human",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Species",
              ": ",
              "Human",
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h3>
                        Beru Whitesun lars
            </h3>
            <button
                        className=""
                        onClick={[Function]}
                        title="Add to Favorites"
            >
                        ★
            </button>
</header>,
          <ul>
            <li>
                        Homeword
                        : 
                        Tatooine
            </li>
            <li>
                        Population
                        : 
                        200000
            </li>
            <li>
                        Species
                        : 
                        Human
            </li>
</ul>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Beru Whitesun lars
</h3>,
              <button
                className=""
                onClick={[Function]}
                title="Add to Favorites"
>
                ★
</button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Beru Whitesun lars",
              },
              "ref": null,
              "rendered": "Beru Whitesun lars",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "★",
                "className": "",
                "onClick": [Function],
                "title": "Add to Favorites",
              },
              "ref": null,
              "rendered": "★",
              "type": "button",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Homeword
                : 
                Tatooine
</li>,
              <li>
                Population
                : 
                200000
</li>,
              <li>
                Species
                : 
                Human
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Homeword",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Homeword",
                  ": ",
                  "Tatooine",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Population",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Population",
                  ": ",
                  "200000",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Population",
                ": ",
                "200000",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Species",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Species",
                  ": ",
                  "Human",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Species",
                ": ",
                "Human",
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Card should match snapshot when rendering a planet card 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card
    addToFav={[Function]}
    card={
        Object {
            "Climate": "murky",
            "Population": "unknown",
            "Residents": "",
            "Terrain": "swamp, jungles",
            "cardType": "planets",
            "favorite": false,
            "name": "Dagobah",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h3>
                    Dagobah
          </h3>
          <button
                    className=""
                    onClick={[Function]}
                    title="Add to Favorites"
          >
                    ★
          </button>
</header>,
        <ul>
          <li>
                    Climate
                    : 
                    murky
          </li>
          <li>
                    Population
                    : 
                    unknown
          </li>
          <li>
                    Residents
                    : 
                    
          </li>
          <li>
                    Terrain
                    : 
                    swamp, jungles
          </li>
</ul>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Dagobah
</h3>,
            <button
              className=""
              onClick={[Function]}
              title="Add to Favorites"
>
              ★
</button>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Dagobah",
            },
            "ref": null,
            "rendered": "Dagobah",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "★",
              "className": "",
              "onClick": [Function],
              "title": "Add to Favorites",
            },
            "ref": null,
            "rendered": "★",
            "type": "button",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Climate
              : 
              murky
</li>,
            <li>
              Population
              : 
              unknown
</li>,
            <li>
              Residents
              : 
              
</li>,
            <li>
              Terrain
              : 
              swamp, jungles
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Climate",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Climate",
                ": ",
                "murky",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Climate",
              ": ",
              "murky",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Population",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Population",
                ": ",
                "unknown",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Population",
              ": ",
              "unknown",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Residents",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Residents",
                ": ",
                "",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Residents",
              ": ",
              "",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Terrain",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Terrain",
                ": ",
                "swamp, jungles",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Terrain",
              ": ",
              "swamp, jungles",
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h3>
                        Dagobah
            </h3>
            <button
                        className=""
                        onClick={[Function]}
                        title="Add to Favorites"
            >
                        ★
            </button>
</header>,
          <ul>
            <li>
                        Climate
                        : 
                        murky
            </li>
            <li>
                        Population
                        : 
                        unknown
            </li>
            <li>
                        Residents
                        : 
                        
            </li>
            <li>
                        Terrain
                        : 
                        swamp, jungles
            </li>
</ul>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Dagobah
</h3>,
              <button
                className=""
                onClick={[Function]}
                title="Add to Favorites"
>
                ★
</button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Dagobah",
              },
              "ref": null,
              "rendered": "Dagobah",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "★",
                "className": "",
                "onClick": [Function],
                "title": "Add to Favorites",
              },
              "ref": null,
              "rendered": "★",
              "type": "button",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Climate
                : 
                murky
</li>,
              <li>
                Population
                : 
                unknown
</li>,
              <li>
                Residents
                : 
                
</li>,
              <li>
                Terrain
                : 
                swamp, jungles
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Climate",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Climate",
                  ": ",
                  "murky",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Climate",
                ": ",
                "murky",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Population",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Population",
                  ": ",
                  "unknown",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Population",
                ": ",
                "unknown",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Residents",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Residents",
                  ": ",
                  "",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Residents",
                ": ",
                "",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Terrain",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Terrain",
                  ": ",
                  "swamp, jungles",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Terrain",
                ": ",
                "swamp, jungles",
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Card should match snapshot when rendering a vehicle card 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card
    addToFav={[Function]}
    card={
        Object {
            "Homeword": "Tatooine",
            "Population": "200000",
            "Species": "Human",
            "cardType": "people",
            "name": "Beru Whitesun lars",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h3>
                    Beru Whitesun lars
          </h3>
          <button
                    className=""
                    onClick={[Function]}
                    title="Add to Favorites"
          >
                    ★
          </button>
</header>,
        <ul>
          <li>
                    Homeword
                    : 
                    Tatooine
          </li>
          <li>
                    Population
                    : 
                    200000
          </li>
          <li>
                    Species
                    : 
                    Human
          </li>
</ul>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Beru Whitesun lars
</h3>,
            <button
              className=""
              onClick={[Function]}
              title="Add to Favorites"
>
              ★
</button>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Beru Whitesun lars",
            },
            "ref": null,
            "rendered": "Beru Whitesun lars",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "★",
              "className": "",
              "onClick": [Function],
              "title": "Add to Favorites",
            },
            "ref": null,
            "rendered": "★",
            "type": "button",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Homeword
              : 
              Tatooine
</li>,
            <li>
              Population
              : 
              200000
</li>,
            <li>
              Species
              : 
              Human
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Homeword",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Homeword",
              ": ",
              "Tatooine",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Population",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Population",
                ": ",
                "200000",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Population",
              ": ",
              "200000",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Species",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Species",
                ": ",
                "Human",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Species",
              ": ",
              "Human",
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h3>
                        Beru Whitesun lars
            </h3>
            <button
                        className=""
                        onClick={[Function]}
                        title="Add to Favorites"
            >
                        ★
            </button>
</header>,
          <ul>
            <li>
                        Homeword
                        : 
                        Tatooine
            </li>
            <li>
                        Population
                        : 
                        200000
            </li>
            <li>
                        Species
                        : 
                        Human
            </li>
</ul>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Beru Whitesun lars
</h3>,
              <button
                className=""
                onClick={[Function]}
                title="Add to Favorites"
>
                ★
</button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Beru Whitesun lars",
              },
              "ref": null,
              "rendered": "Beru Whitesun lars",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "★",
                "className": "",
                "onClick": [Function],
                "title": "Add to Favorites",
              },
              "ref": null,
              "rendered": "★",
              "type": "button",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Homeword
                : 
                Tatooine
</li>,
              <li>
                Population
                : 
                200000
</li>,
              <li>
                Species
                : 
                Human
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Homeword",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Homeword",
                  ": ",
                  "Tatooine",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Population",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Population",
                  ": ",
                  "200000",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Population",
                ": ",
                "200000",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Species",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Species",
                  ": ",
                  "Human",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Species",
                ": ",
                "Human",
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Card should match snapshot when the card is favorited 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card
    addToFav={[Function]}
    card={
        Object {
            "Homeword": "Tatooine",
            "Population": "200000",
            "Species": "Human",
            "cardType": "people",
            "favorite": true,
            "name": "Beru Whitesun lars",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h3>
                    Beru Whitesun lars
          </h3>
          <button
                    className="favorite"
                    onClick={[Function]}
                    title="Add to Favorites"
          >
                    ★
          </button>
</header>,
        <ul>
          <li>
                    Homeword
                    : 
                    Tatooine
          </li>
          <li>
                    Population
                    : 
                    200000
          </li>
          <li>
                    Species
                    : 
                    Human
          </li>
</ul>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Beru Whitesun lars
</h3>,
            <button
              className="favorite"
              onClick={[Function]}
              title="Add to Favorites"
>
              ★
</button>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Beru Whitesun lars",
            },
            "ref": null,
            "rendered": "Beru Whitesun lars",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "★",
              "className": "favorite",
              "onClick": [Function],
              "title": "Add to Favorites",
            },
            "ref": null,
            "rendered": "★",
            "type": "button",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              Homeword
              : 
              Tatooine
</li>,
            <li>
              Population
              : 
              200000
</li>,
            <li>
              Species
              : 
              Human
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Homeword",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Homeword",
              ": ",
              "Tatooine",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Population",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Population",
                ": ",
                "200000",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Population",
              ": ",
              "200000",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "Species",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Species",
                ": ",
                "Human",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Species",
              ": ",
              "Human",
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h3>
                        Beru Whitesun lars
            </h3>
            <button
                        className="favorite"
                        onClick={[Function]}
                        title="Add to Favorites"
            >
                        ★
            </button>
</header>,
          <ul>
            <li>
                        Homeword
                        : 
                        Tatooine
            </li>
            <li>
                        Population
                        : 
                        200000
            </li>
            <li>
                        Species
                        : 
                        Human
            </li>
</ul>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Beru Whitesun lars
</h3>,
              <button
                className="favorite"
                onClick={[Function]}
                title="Add to Favorites"
>
                ★
</button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Beru Whitesun lars",
              },
              "ref": null,
              "rendered": "Beru Whitesun lars",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "★",
                "className": "favorite",
                "onClick": [Function],
                "title": "Add to Favorites",
              },
              "ref": null,
              "rendered": "★",
              "type": "button",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                Homeword
                : 
                Tatooine
</li>,
              <li>
                Population
                : 
                200000
</li>,
              <li>
                Species
                : 
                Human
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Homeword",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Homeword",
                  ": ",
                  "Tatooine",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Homeword",
                ": ",
                "Tatooine",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Population",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Population",
                  ": ",
                  "200000",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Population",
                ": ",
                "200000",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": "Species",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Species",
                  ": ",
                  "Human",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Species",
                ": ",
                "Human",
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
